resonantLimits/combine_res_all.sh

```diff
@@ -6,21 +6,23 @@ declare -a MASSES;
 TAG=""
 VAR="DNNoutSM_kl_1"
 SIGNAL="ggFRadion"
+BASEDIR="/home/llr/cms/${USER}/CMSSW_11_1_9/src/KLUBAnalysis"
```

Comment:
Wouldn't `pwd` be a better default?

Reply:
I agree the current version is not ideal. However, using ```pwd``` would not be strict enough, breaking as soon as the user decides to run the code from an unconventional directory. Additionally, I think it is good to be explicit on which CMSSW release to use, and in which folder the code is expected to be run. I've thus replaced the above by ```${HOME}/CMSSW_11_1_9/src/KLUBAnalysis```; it will likely also work for people outside LLR.